scripts/common/cmdline_helpers.py

```diff
@@ -37,15 +37,21 @@ def f(*args, **kwargs):
     return tmp_dir_decorator
 
 
-def solc_bin_report(solc_binary: str, input_files: List[Path], via_ir: bool) -> FileReport:
+def solc_bin_report(solc_binary: str,
+                    input_files: List[Path],
+                    via_ir: bool,
+                    optimize: bool = False,
+                    yul_optimizations: Optional[str] = None) -> FileReport:
```

Comment:
Just as a note, so we can remember to eventually refactor this. We could pass a dictionary here like  what was done in https://github.com/ethereum/solidity/blob/bbb7f58be026fdc51b0b4694a6f25c22a1425586/scripts/externalTests/test_helpers.py#L52 or event have a class for compiler settings so we could reuse in other scripts. I believe this will be required eventually when we migrate the external tests to python.